src/Proto.Remote/Endpoints/EndpointManager.cs

```diff
@@ -196,7 +196,7 @@ internal IEndpoint GetClientEndpoint(string systemId)
     private void SpawnActivator()
     {
         var props = Props.FromProducer(() => new Activator(_remoteConfig, _system));
-        ActivatorPid = _system.Root.SpawnNamedSystem(props, "activator");
+        ActivatorPid = _system.Root.SpawnNamedSystem(props, "$activator");
```

Comment:
I see that this change breaks the remoting tests as other places still rely on the old name.
The name could be turned into a const as for the other named actors above, e.g. $server_seed and $client_seed

Reply:
I didn't get to run the tests on my machine as I don't have the 3.1 sdk 😃 I can fix that. Though it's not comforting knowing that there are such "couplings" in the system with dependencies on actor names. Have it been a discussion point to reduce that implicit coupling by either introducing an abstraction between or having something akin to a centralized "name registry" (don't think I like this 😀 ), or something completely different

Reply:
[82187d7](https://github.com/asynkron/protoactor-dotnet/pull/1649/commits/82187d75f8c571cab0383940087e223bb2c9ace1) This should resolve it